README.md

```diff
@@ -87,4 +95,8 @@ The most common mood is :-)
 
 **think**: Should you use an array or a hash to solve this problem? Why?
 
+I used a hash of arrays because I wanted to access collected values by total, and this way I can call a key to see which one holds the most values.
+
 **think**: What if we eventually want to add feelings to our analysis? Can we write this code in a way that will prevent us from having to re-write it later?
+
+I *believe* I have written code that will allow me to simply slot in more moods and smileys and then continue to run as is.
```

Comment:
I don't think this is true. It's definitely not difficult to add more moods to your code, because it's readable. But this method actually could have been written in a way that adding more moods would not have required this method to be rewritten at all.